src/Control/Email/Email.php

```diff
@@ -398,26 +398,21 @@ public function removeData(string $name)
         return $this;
     }
 
-    public function getHTMLTemplate(): string
+    public function getHTMLTemplate(): string|array
     {
         if ($this->HTMLTemplate) {
             return $this->HTMLTemplate;
         }
 
-        return ThemeResourceLoader::inst()->findTemplate(
-            SSViewer::get_templates_by_class(static::class, '', Email::class),
-            SSViewer::get_themes()
-        );
+        return SSViewer::get_templates_by_class(static::class, '', Email::class);
     }
 
     /**
-     * Set the template to render the email with
+     * Set the template to render the email with.
+     * Do not include a file extension unless you are referencing a full absolute file path.
      */
     public function setHTMLTemplate(string $template): static
     {
-        if (substr($template ?? '', -3) == '.ss') {
-            $template = substr($template ?? '', 0, -3);
-        }
```

Comment:
We don't know what template engine will be used or what file extension(s) it supports - better to just not do any normalisation.